.github/workflows/ci.yml

```diff
@@ -8,14 +8,22 @@ on:
     branches:
       - master
   workflow_dispatch:
+    inputs:
+      splits:
+        description: 'The number of splits for test_main'
+        required: true
+        default: '10'
```

Comment:
Curious, why default to 10 but then override to 14 below ?

Reply:
These are all just numbers at this point. 🤷 I'll change it to 14.